src/api/products/products.repository.js

```diff
@@ -15,11 +16,29 @@ async function getAll({ skip, limit }) {
 
 async function getById({ id }) {
   const user = await productsModel.findById(id).lean();
-  console.log('user', user);
   return user;
 }
 
+async function getLastOrderProducts({ orderId }) {
```

Comment:
Is this function ever called? I think it might be redundant.

Reply:
Nope, you are right...I created it at the beginning but then realized it was unnecessary and I forgot to delete it